frontend/src/store/modules/website/mutations.js

```diff
@@ -14,41 +17,41 @@ export default {
             ...data,
         };
     },
-    [SET_CURRENT_WEBSITE]: (state, website) => {
+    [SET_CURRENT_WEBSITE]: (state) => {
         state.isCurrentWebsite = true;
-        state.currentWebsite = website;
     },
     [UPDATE_CURRENT_WEBSITE]: (state, website) => {
         state.currentWebsite = {
             ...state.currentWebsite,
             ...website
         };
     },
-    [RESET_CURRENT_WEBSITE]: (state) => {
-        state.currentWebsite = {
-            name: '',
-            domain: '',
-            single_page: false,
-            tracking_number: ''
-        };
+    [RESET_CURRENT_WEBSITES]: (state) => {
+        state.relateUserWebsites = [];
+        state.selectedWebsite = {};
         state.isCurrentWebsite = false;
     },
     [SET_FETCH_TRUE]: (state) => {
         state.isFetchedWebsite = true;
     },
-    [RESET_WEBSITE_DATA]: (state) => {
+    [RESET_WEBSITES_DATA]: (state) => {
         state.newWebsite = {
             name: '',
             domain: '',
             single_page: false,
         };
-        state.currentWebsite = {
-            name: '',
-            domain: '',
-            single_page: false,
-            tracking_number: ''
-        };
+        state.relateUserWebsites = [];
+        state.selectedWebsite = undefined;
         state.isCurrentWebsite = false;
         state.isFetchedWebsite = false;
-    }
+    },
+    [SET_SELECTED_WEBSITE]: (state, website) => {
+        state.selectedWebsite = website;
```

Comment:
I'm not sure sure if you need to make a snapshot of selected website. You can simply store ID and in GET_SELECTED_WEBSITE find website object based on it